src/deepsparse/loggers/python_logger.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Implementation of the Python Logger that logs to the stdout
+"""
+from datetime import datetime
+from typing import Any
+
+from deepsparse.loggers import BaseLogger, MetricCategories
+
+
+__all__ = ["PythonLogger"]
+
+
+class PythonLogger(BaseLogger):
```

Comment:
I still think `StdoutLogger` would be more clear as to what this class does (PythonLogger also makes it sound like its using the logging package), but I know we have PythonLogger in sparseml, so consistency is probably better? 🤷